lib/src/zap/zap.h

```diff
@@ -554,6 +559,37 @@ zap_err_t zap_close(zap_ep_t ep);
  */
 zap_err_t zap_send(zap_ep_t ep, void *buf, size_t sz);
 
+/**
+ * \brief Send data to peer using map.
+ *
+ * This is a more efficient send interface comparing to \c zap_send(). The data
+ * to be sent is described by \c map, \c buf and \c len. The data will not be
+ * copied out to the internal zap buffer (unlike \c zap_send()) and the
+ * application should not modify the data in the specified range (buf[0..len-1])
+ * before the send operation is completed. The completion of the send operation
```

Comment:
... The \c buf buffer is owned by the transport until the ZAP_SEND_MAPPED_COMPLETE event is received..,

Reply:
I added the sentence. Now it reads

... the send operation is completed. _The \c buf buffer is owned ..... is received._ The completion of the send ....